v2/thirdparty/serverless/with-aws-lambda/appsync-integration.mdx

```diff
@@ -0,0 +1,41 @@
+---
+id: appsync-integration
+title: AppSync integration
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/serverless/with-aws-lambda/appsync-integration.mdx -->
+
+# Lambda Authorizer
+
+A Lambda Authorizer configured like in the [Authorizer guide](./authorizer) can help integrate Supertokens with an AppSync service.  
+
+## 1) Set up the AppSync service
+
+Set up the AppSync service with an API key authorization. For more details, please see the [AWS documentation](https://docs.aws.amazon.com/appsync/latest/devguide/quickstart.html).
+
+## 2) Configure the API Gateway with the authorizer
+
+Follow the [Authorizer guide](./authorizer) to set up the API Gateway with the `/auth`, and `/graphql` resources set up.
```

Comment:
Specify what should one do with the `/auth` gateway (point it to to a lambda function that contains the auth APIs shown in step 3)